### DISCUSSION TITLE
Falied to install wazuh

### DISCUSSION THREAD

**Discussion by Saifullahil-Mazid** [wazuh/wazuh#25640] [Q&A]
Hi. I've been using wazuh from last year. I was trying to install wazuh in a new vm ubuntu 22.04.4 lts but it falied by repeating an error. the installation continued till filebeat but stopped eventually. I was trying to quickstart. What is the possible problem?

**Comment by MiguelazoDS** [comment]
Hi @Saifullahil-Mazid 

What version are you trying to install? what is the error?

**Comment by sebasfalcone** [comment]
Inactive thread, please re-open it if necessary
